lisa/wlgen/rta.py

```diff
@@ -329,23 +329,39 @@ def _calibrate(cls, target, res_dir):
         logger = cls.get_logger()
 
         # Create calibration task
-        max_rtprio = int(target.execute('ulimit -Hr').split('\r')[0])
-        logger.debug('Max RT prio: %d', max_rtprio)
+        if target.is_rooted:
+            max_rtprio = int(target.execute('ulimit -Hr').split('\r')[0])
+            logger.debug('Max RT prio: %d', max_rtprio)
 
-        if max_rtprio > 10:
-            max_rtprio = 10
+            if max_rtprio > 10:
+                max_rtprio = 10
+
+            sched = {
```

Comment:
Urgh, the docstring is out of date - I killed the `sched` dict and split it into `sched_policy` and `priority` parameters.

Reply:
I added a commit to fix the docstring